templates/traas.yaml

```diff
@@ -107,18 +131,50 @@ parameters:
     default: master
     description: Release to be used by --release quickstart option
 
-
 resources:
 
+  sg_uc:
+    type: OS::Neutron::SecurityGroup
+    properties:
+      name: sg_uc
+      description: Sec group for UC
+      rules:
+      - protocol: icmp
+      - direction: ingress
+        protocol: udp
```

Comment:
technically, these may restrict everything but SSH and ICMP, and may be, DNS traffic.
And CI jobs' vxlan & bridging tricks will just keep working as we split the admin from cluster networks.